### ISSUE TITLE
Feature request: add option for opacity and 

### ISSUE THREAD

**Issue by hanh45** [mpalourdio/ng-http-loader#59]
Dear @mpalourdio ,

Current i see default of opacity in master CSS with #spinner is 0.7
Can you add option for opacity? Ex: 
```html
<spinner 
    [backgroundColor]="'#ff0000'"
    [spinner]="spinkit.skWave"
    [debounceDelay]="200"
    [opacity]="1"
>
</spinner>
```

And, i see "spinkit.skWave" display don't like "wave" (not same for space of | ) in https://github.com/tobiasahlin/SpinKit
Can you repair CSS of this?

Thanks.

**Comment by mpalourdio** [comment]
I don't think that I will implement the opacity at component level, it's very easy to do it at css level (browse issues to see how to do it, it has been asked multiple times)

For the spinkit.skWave, you're right, I've just checked and it's broken. Something is happening during the module bundling. I have quickly checked and I can't find what's wrong, but it definitely needs to be fixed. Thanks for the report !

Will try to fix later.

**Comment by hanh45** [comment]
Thanks for quick response,
1. For opacity option, i know can override with my CSS but if have it, i think very good with production of you because everybody don't customize when use.
2. With spinkit.skWave, I tested on FF, IE, Chrome and same as reported. Hopefully there will be an edit soon.

**Comment by mpalourdio** [comment]
Fixed in v0.9.1, please see https://github.com/mpalourdio/ng-http-loader/blob/master/CHANGELOG.md#v091

Thanks for feedback !
